Assets/Scripts/PlayerInteraction/AnimalMovement.cs

```diff
@@ -64,6 +65,12 @@ public override void OnInspectorGUI()
                 movementScript.validTeleportLayers = FieldToLayerMask(teleportLayersSelected);
            
             }
+            
+            if (GUI.changed)
```

Comment:
What does this do?

Reply:
This part of the script is a custom editor so that the inspector responds to what you click/add to the vision effects. This makes sure that if the inspector values have changed, the values are saved (ran into the bug before where I was changing values in the inspector and on play, the values reverted back to the original)